### ISSUE TITLE
Implement SNN in Keras

### ISSUE THREAD

**Issue by guyko81** [fchollet/keras#6947]
Hi,

can you implement the new activation from [Self-Normalizing Neural Networks paper](https://arxiv.org/pdf/1706.02515.pdf)? There's a Tensorflow implementation [here,](https://github.com/bioinf-jku/SNNs) but I only have Keras codes... 
 :) 

thanks

**Comment by Dref360** [comment]
SELU is already in a PR and the dropout aswell.

**Comment by guyko81** [comment]
already? that's so good news! thank you

**Comment by Dref360** [comment]
Please close your issue :)

**Comment by guyko81** [comment]
Sure, sorry :)
